apps/desktop-wallet/src/api/transactions.ts

```diff
@@ -35,7 +35,7 @@ export const buildSweepTransactions = async (fromAddress: Address, toAddressHash
 }
 
 export const signAndSendTransaction = async (fromAddress: Address, txId: string, unsignedTx: string) => {
-  const signature = transactionSign(txId, fromAddress.privateKey)
+  const signature = keyring.signTransaction(txId, fromAddress.hash)
```

Comment:
Since we are not storing private keys on our side anymore, the signing of a transaction is delegated to the keyring, like on Metamask: https://github.com/MetaMask/eth-hd-keyring/blob/main/index.js#L170